src/app/pages/my-virtrolio/my-virtrolio.component.ts

```diff
@@ -46,6 +57,14 @@ export class MyVirtrolioComponent implements OnInit {
     this.copyButtonText = 'Copied!';
   }
 
+  /**
+   * Closes sharing link modal and opens up desktop sharing modal
+   */
+  openDesktopSharing() {
```

Comment:
Method name should be changed as this isn't specifically "Desktop". Maybe `openShareLinkSocialModal` or `openShareLinkModal`